apps/prairielearn/src/lib/preact.tsx

```diff
@@ -52,12 +52,12 @@ export function hydrate<T>(content: VNode<T>, nameOverride?: string): VNode {
     throw new Error('hydrate expects a Preact component');
   }
 
-  if (!nameOverride && !Component.displayName && !Component.name) {
+  const componentName = nameOverride || Component.name || Component.displayName;
```

Comment:
For some reason, fragments got the letter `m` as their name, so I excluded that.